src/js/components/sorter/Sorter.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { ButtonToolbar, ToggleButtonGroup, ToggleButton } from 'react-bootstrap';
+import './sorter.style.less';
+
+class Sorter extends Component {
+
+  state = {
```

Comment:
if you want to set your initial state basing on props you need to use `constructor` and call `super`, only if you're not relying on props at initial state you can omit those

Reply:
it's working now. why?

Reply:
it'll be working (as your props are declared and just passed) but it's a case for good practices to set this kind of stuff inside constructor (as it's the right place to set everything before use) - in terms of super and constructor: https://overreacted.io/why-do-we-write-super-props/